sale_promotion_domain_product_discount/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
It's a common guideline to add copyright in the relevant code files. Something like:

```suggestion
# Copyright 2022 Cemtix - Ivan Sokolov
# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
{
```